docs/compatibility.md

```diff
@@ -104,21 +104,21 @@
 | Method | Supported |
 |--------|-----------|
 | allowsNull | :heavy_check_mark: Yes |
-| canBePassedByValue | todo |
-| getClass | todo |
+| canBePassedByValue | :heavy_check_mark: Yes |
+| getClass | :x: No - could currently cause recursion |
```

Comment:
I've created new issue #13 to investigate this - for now I'm happy to say it's currently unsupported and implement later.